en/docs/publish/api-microgateway/consul-service-discovery.md

```diff
@@ -0,0 +1,91 @@
+# Consul Service Discovery
+
+WSO2 Envoy based Microgateway supports [Consul](https://www.hashicorp.com/products/consul) service discovery to read from the Consul service catalog and discover upstreams automatically.
```

Comment:
Lets remove envoy based part
And also provide mention like
WSO2 Microgateway supports service discovery by connecting to consul service registry and discover upstream services automatically

Reply:
fixed 6b432b0